.travis.yml

```diff
@@ -0,0 +1,34 @@
+dist: trusty
+sudo: required
```

Comment:
Do you actually need sudo?
You can run Bazel on Container-based infrastructure if upgrade g++ and libstdc++ though.  
https://github.com/yugui/jsonnetunit/blob/master/.travis.yml#L24